projects/fmcjesdadc1/src/app/fmcjesdadc1.c

```diff
@@ -256,19 +329,27 @@ int main(void)
 		printf("axi_jesd204_rx_status_read() error: %"PRIi32"\n", status);
 	}
 
+	//ADC output_format
+	ad9250_output_format(ad9250_0_device, 0);
+	ad9250_output_format(ad9250_1_device, 0);
+
 	// PRBS test
-	ad9250_test(ad9250_0_device, AD9250_TEST_PNLONG);
+	ad9250_test_mode(ad9250_0_device, 5);
+	ad9250_transfer(ad9250_0_device);
 	if(axi_adc_pn_mon(ad9250_0_core, AXI_ADC_PN23, 10) == -1) {
 		printf("%s ad9250_0 - PN23 sequence mismatch!\n", __func__);
 	};
-	ad9250_test(ad9250_1_device, AD9250_TEST_PNLONG);
+	ad9250_test_mode(ad9250_1_device, 5);
+	ad9250_transfer(ad9250_1_device);
 	if(axi_adc_pn_mon(ad9250_1_core, AXI_ADC_PN23, 10) == -1) {
 		printf("%s ad9250_1 - PN23 sequence mismatch!\n", __func__);
 	};
 
 	// set up ramp output
-	ad9250_test(ad9250_0_device, AD9250_TEST_RAMP);
```

Comment:
You could keep the macros for the test modes.

Reply:
I agree. But the modification you suggest implies updating the ad9250 driver, while the PR handles the fmcjesdadc1 project update. I will create a separate commit/PR afterwards.